src/cascadia/TerminalApp/Tab.cpp

```diff
@@ -111,10 +111,12 @@ std::optional<GUID> Tab::GetFocusedProfile() const noexcept
 // - control: reference to the TermControl object to bind event to
 // Return Value:
 // - <none>
-void Tab::BindEventHandlers(const TermControl& control) noexcept
+void Tab::BindEventHandlers() noexcept
 {
-    _AttachEventHandlersToPane(_rootPane);
-    _AttachEventHandlersToControl(control);
+    const auto rootPaneAsLeaf = std::dynamic_pointer_cast<LeafPane>(_rootPane);

```

Comment:
Doesn't this technically need `dynamic_cast` to work which would require RTTI to be turned on?

I saw @zadjii-msft was going to turn it on for the arg parser in that PR (#4023).

Reply:
Yup, that's RTTI

Reply:
OK.